examples/stats/basic_prototype.py

```diff
@@ -29,10 +29,11 @@ def main():
     boundaries = [0, 1/16, 1/32]
 
     video_size = measure.BaseMeasure("my.org/measure/video_size", "size of processed videos", "MBy")
-    my_view = view.View("my.org/views/video_size_cum", "processed video size over time", video_size, aggregation.DistributionAggregation(boundaries))
+    my_view = view.View("my.org/views/video_size_cum", "processed video size over time", [], video_size, aggregation.DistributionAggregation(boundaries))
```

Comment:
Put each parameter on a single line to avoid line too long,
```
my_view = view.View(
    "my.org/views/video_size_cum",
    "processed video size over time",
    [],
    video_size,
    aggregation.DistributionAggregation(boundaries))
```